testsuite/dockerfiles/server-all-in-one/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM registry.suse.com/bci/bci-init:15.4
+FROM registry.suse.com/bci/bci-init:15.5
 # from https://raw.githubusercontent.com/rjmateus/uyuni-container/main/uyuni/Dockerfile
 
 RUN zypper ar -G http://download.opensuse.org/update/leap/15.4/backports/ backports_update_repo
```

```suggestion
RUN zypper ar -G http://download.opensuse.org/update/leap/15.5/backports/ backports_update_repo
```
and the same applies to the  other repos mentioning 15.4  instead of 15.5
~~~
RUN zypper ar -G http://download.opensuse.org/distribution/leap/15.4/repo/oss/ os_pool_repo
RUN zypper ar -G http://download.opensuse.org/update/leap/15.4/oss/ os_update_repo
RUN zypper ar -G 
RUN zypper ar -G http://download.opensuse.org/update/leap/15.4/sle/ sle_update_repo
~~~
Sorry, I can't add  a suggestion for  changing also those